src/test/java/com/stripe/functional/TelemetryTest.java

```diff
@@ -52,18 +54,30 @@ public void testTelemetryEnabled() throws StripeException, IOException, Interrup
     Balance.retrieve();
     RecordedRequest request2 = server.takeRequest();
     String telemetry1 = request2.getHeader("X-Stripe-Client-Telemetry");
-    ClientTelemetryPayload payload1 =
-        ApiResource.GSON.fromJson(telemetry1, ClientTelemetryPayload.class);
-    assertEquals(payload1.lastRequestMetrics.requestId, "req_1");
-    assertTrue(payload1.lastRequestMetrics.requestDurationMs > 30);
+    JsonObject requestMetrics1 =
+        jsonParser
+            .parse(telemetry1)
+            .getAsJsonObject()
+            .get("last_request_metrics")
+            .getAsJsonObject();
+    String requestId1 = requestMetrics1.get("request_id").getAsString();
+    Long requestDurationMs1 = requestMetrics1.get("request_duration_ms").getAsLong();
```

Comment:
This is a little bit more awkward because we can no longer deserialize the JSON string into a `ClientTelemetryPayload` instance so we're using a raw `JsonObject` instead.